src/libs/SearchUIUtils.ts

```diff
@@ -276,7 +276,7 @@ function isAmountTooLong(amount: number, maxLength = 8): boolean {
 
 function isTransactionAmountTooLong(transactionItem: TransactionListItemType | SearchTransaction | OnyxTypes.Transaction) {
     // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
-    const amount = Math.abs(transactionItem.modifiedAmount || transactionItem.amount);
+    const amount = Math.abs(transactionItem.modifiedAmount ?? transactionItem.amount);
```

Comment:
```suggestion
    const amount = Math.abs(transactionItem.modifiedAmount || transactionItem.amount);
```

Looks like some transactions use default of 0 in modifiedAmount (which does not seem right but we have to take into account for now)